### PR TITLE
[4.x] translate dimension conjunction

### PR DESCRIPTION
**Pull request by peimn**
We need this translation in some languages

### CODE REVIEW

**Comment by jasonvarga** [comment]
Please merge `4.x` into this branch.

**Comment by peimn** [comment]
Done!

File: resources/lang/en/messages.php [hunk 1]
```diff
@@ -241,4 +241,10 @@
     'user_wizard_roles_groups_intro' => 'Users can be assigned to roles that customize their permissions, access, and abilities throughout the Control Panel.',
     'user_wizard_super_admin_instructions' => 'Super admins have complete control and access to everything in the control panel. Grant this role wisely.',
     'view_more_count' => 'View :count more',
+    'units.B' => ':count B',
+    'units.KB' => ':count KB',
+    'units.MB' => ':count MB',
+    'units.GB' => ':count GB',
+    'units.s' => ':counts',
+    'units.ms' => ':countms',
```
Comment (jasonvarga) [inline resources/lang/en/messages.php:249]: Please remove these lines. They already exist at line 218.

**Comment by jasonvarga** [comment]
Just curious, how would you display dimensions in your language?

**Comment by peimn** [comment]
۳۰۰ در ۴۰۰

**Comment by jasonvarga** [comment]
Thanks!

**Comment by peimn** [comment]
That was for Persian. 
For Arabic that is:
۳۰۰ فی ۴۰۰

**Pull request merged**